api/src/views/lottery-released.hbs

```diff
@@ -0,0 +1,53 @@
+{{#> layout_default }}
+    <h1>
+        {{t "lotteryReleased.header" appOptions}}
+    </h1>
+    <table role="presentation" border="0" cellpadding="0" cellspacing="0">
+        <tbody>
+        <tr>
+            <td align="left">
+            <p>
+                {{t "t.hello"}},
+                <br/><br/>
+                {{t "lotteryReleased.adminApprovedStart" appOptions}} <a href="{{ appUrl }}">{{t "t.partnersPortal"}}</a> {{t "lotteryReleased.adminApprovedEnd"}}
+                <br/><br/>
+                  <strong>{{t "requestApproval.accessListing"}}</strong>
+                <br/><br/>
+            </p>
+            </td>
+        </tr>
+        </tbody>
+    </table>
+    <table role="presentation" border="0" cellpadding="0" cellspacing="0">
+        <tbody>
+        <tr>
+            <td align="center">
+            <a
```

Comment:
I probably just missed it, but I don't see the button in the copy so don't know what it is supposed to look like. The "View Listing" button seems to be lower down than I'd expect. In other words there is a bigger gap between the text above it and the "Thank you" below it. Don't know if this is intended
<img width="519" alt="image" src="https://github.com/bloom-housing/bloom/assets/42942267/6b962071-fa6a-4479-bb85-25572150e829">

Reply:
This is just copy/pasta format of our current approval emails which was the request from product (sorry I didn't mention that!)